h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -70,6 +70,7 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
             "cold_start", // if true, will start GLM model from initial values and conditions
             "lambda_min_ratio",
             "beta_constraints",
+            "expose_constraints",
```

Comment:
I don't think you should expose internal parameters through the rest API.

Reply:
Removed.